internal/timestamp/timestamp.go

```diff
@@ -43,16 +43,8 @@ func Timestamp(req *signature.SignRequest, opts tspclient.RequestOptions) ([]byt
 	if err != nil {
 		return nil, err
 	}
-	info, err := token.Info()
-	if err != nil {
-		return nil, err
-	}
-	timestamp, err := info.Validate(opts.Content)
-	if err != nil {
-		return nil, err
-	}
 	tsaCertChain, err := token.Verify(ctx, x509.VerifyOptions{
-		CurrentTime: timestamp.Value,
+		CurrentTime: req.SigningTime,
```

Comment:
We should remain the previous implementation due to security consideration we have discussed offline.

Besides, we should have a sanity check on the `req.SigningTime` and the `timestamp.Value` as it only makes sense if the `req.SigningTime` is after the `timestamp.Value`.

Reply:
Updated.